ts/src/test/static/markets/upbit.json

```diff
@@ -0,0 +1,9869 @@
+{
```

Comment:
@sc0Vu we don't want to store all the markets of the exchange, just a few we want to keep things light and fast.